actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Add `-g` and `-c` (shorts for _grep_ and _controller_ respectively) options
+    for `rake routes` task. This options returns the url `name`, `verb`, `path`
+    and `reqs` field that match the pattern or returns routers for specific
+    controller.
+
+    See #18902.
+
+    *Anton Davydov*
```

Comment:
You're also deprecating `CONTROLLER` please mention that, and how to replace them. Perhaps you can find some inspiration in: https://github.com/rails/rails/pull/22173/files#diff-1846d4882fa63eb5b1271541995cbfdcR1 😁